eth-contracts/Dockerfile

```diff
@@ -12,10 +12,12 @@ COPY . .
 
 RUN npm run postinstall
 
+RUN ./scripts/setup-predeployed-ganache.sh /usr/db 1000000000000
```

Comment:
Are the docker changes part of this fix? Didn't see that in the description.

Reply:
@phelpsdb for review of this part!